.ebextensions/efs-mount.config

```diff
@@ -85,8 +85,8 @@ files:
 
         mountpoint -q ${EFS_MOUNT_DIR}
         if [ $? -ne 0 ]; then
-            echo "mount -t nfs4 -o nfsvers=4.1,rsize=1048576,wsize=1048576,hard,timeo=600,retrans=2 ${EFS_FILE_SYSTEM_ID}.efs.${EFS_REGION}.amazonaws.com:/ ${EFS_MOUNT_DIR}"
-            mount -t nfs4 -o nfsvers=4.1,rsize=1048576,wsize=1048576,hard,timeo=600,retrans=2 ${EFS_FILE_SYSTEM_ID}.efs.${EFS_REGION}.amazonaws.com:/ ${EFS_MOUNT_DIR}
+            echo "mount -t nfs4 -o nfsvers=4.1,rsize=1048576,wsize=1048576,hard,timeo=600,retrans=2,noresvport ${EFS_FILE_SYSTEM_ID}.efs.${EFS_REGION}.amazonaws.com:/ ${EFS_MOUNT_DIR}"
+            mount -t nfs4 -o nfsvers=4.1,rsize=1048576,wsize=1048576,hard,timeo=600,retrans=2,noresvport ${EFS_FILE_SYSTEM_ID}.efs.${EFS_REGION}.amazonaws.com:/ ${EFS_MOUNT_DIR}
```

Comment:
LGTM. Command checks out with what is shown in our [AWS EFS page](https://ap-southeast-1.console.aws.amazon.com/efs?region=ap-southeast-1#/file-systems/fs-5ada251b) and [AWS doc](https://docs.aws.amazon.com/efs/latest/ug/mounting-fs-mount-cmd-ip-addr.html). 

What are your test plans to check if its working in prod?

Reply:
I don't really have a test plan to check if the remounting doesn't work. 

I do have a regression test.
The deployment through EB should work -- tested by simulating a mounting [failure](https://github.com/opengovsg/FormSG/commit/4d050cb0da8da6237dcc0d2433306ee64069f51d) which results in EC2 mount failure and thus EB abort the deployment of new image

Reply:
Regression test sounds sufficient. Thanks